actions/create-pr/src/create-commit.ts

```diff
@@ -20,13 +19,10 @@ export function createCommit({
   authorName,
   authorEmail,
 }: CreateCommitOptions): void {
-  const randomSuffix = generateRandomSuffix(6);
```

Comment:
This was changing the branch name, so it didn't match the branch name we were using to create the PR. I've moved it up one level now so we can use the same branch name for both tasks.